exporter/jaeger/jaegerthrifthttpexporter/exporter_test.go

```diff
@@ -20,33 +20,26 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 
 	"github.com/open-telemetry/opentelemetry-collector/config/configmodels"
 	"github.com/open-telemetry/opentelemetry-collector/consumer/consumerdata"
 )
 
-func TestNew(t *testing.T) {
-	const testHTTPAddress = "http://a.test.dom:123/at/some/path"
+const testHTTPAddress = "http://a.test.dom:123/at/some/path"
 
-	type args struct {
-		config      configmodels.Exporter
-		httpAddress string
-		headers     map[string]string
-		timeout     time.Duration
-	}
+type args struct {
+	config      configmodels.Exporter
+	httpAddress string
+	headers     map[string]string
+	timeout     time.Duration
+}
+
+func TestNew(t *testing.T) {
 	tests := []struct {
```

Comment:
This one also is a single test case, let's remove the `tests`. It can be added later if there are multiple similar tests.